clang/test/dpct/sharedmem-dim-noUSM.cu

```diff
@@ -82,10 +82,18 @@ __global__ void kernel3() {
   __shared__ int a[sizeof(float3) * 3];
 }
 
+// CHECK: void foo3() {
+// CHECK: dpct::get_default_queue().submit(
+// CHECK-NEXT: [&](sycl::handler &cgh) {
+// CHECK: /*
+// CHECK-NEXT: DPCT1083:{{[0-9]+}}: The size of local memory in the migrated code may be different from the original code. Check that the allocated memory size in the migrated code is correct.
+// CHECK-NEXT: */
+// CHECK: /*
+// CHECK-NEXT: DPCT1106:{{[0-9]+}}: The size of sycl::float3 is 16 bytes while in original code size of float3 is 12 bytes. You may need to adjust the code.
```

Comment:
Seems like 2 warnings are about the same: DPCT1083 and DPCT1106. Should they be merged somehow?

Reply:
they are same type, while 1106 is specific to "float3"

Reply:
```
// CHECK: void test_shared() {
// CHECK: dpct::get_default_queue().submit(
// CHECK-NEXT: [&](sycl::handler &cgh) {
// CHECK: /*
// CHECK-NEXT: DPCT1083:{{[0-9]+}}: The result of sizeof operator in the migrated code may be different from the original code. Check that the size in the migrated code is correct.
// CHECK-NEXT: */
// CHECK-NEXT: sycl::local_accessor<int, 1> a_acc_ct1(sycl::range<1>(sizeof(sycl::float3) * 3), cgh);
// CHECK: });
// CHECK: }
void test_shared() {
  kernel<<<1, 1>>>();
}
```
I try to merge these two warnings, WDYT ?